activation/wire/wire_v2.go

```diff
@@ -0,0 +1,76 @@
+package wire
+
+import "github.com/spacemeshos/go-spacemesh/common/types"
+
+//go:generate scalegen
+
+type ActivationTxV2 struct {
+	InnerActivationTxV2
+
+	SmesherID types.NodeID
+	// Signature over InnerActivationTxV2
```

Comment:
Maybe we should add as information that it is the root of the merkle tree formed from all fields of `InnerActiationTxV2`?

Reply:
Yeah, it's worth documenting it when we decide on the signing schema.